File: src/gestorAplicacion/Clientes/Animal.java
package gestorAplicacion.Clientes;

public  class Animal {
	final static double valorsobrecargo=2000;
	public Cliente duenno;
	public Cliente getDuenno() {
		return duenno;
	}

	public void setDuenno(Cliente duenno) {
		this.duenno = duenno;
	}

	public int edad;
	public int getEdad() {
		return edad;
	}

	public void setEdad(int edad) {
		this.edad = edad;
	}

	public String nombreMascota;

	
	public String getNombreMascota() {
		return nombreMascota;
	}

	public void setNombreMascota(String nombreMascota) {
		this.nombreMascota = nombreMascota;
	}


	
	public Animal(Cliente duenno, int edad, String nombre){
		this.duenno=duenno;
		this.edad=edad;
		this.nombreMascota= nombre;

	}
	public double Sobrecargo(){
		double valor= this.edad*Animal.valorsobrecargo;
		return valor;
	}

}

File: Python/gestorAplicacion/Clientes/Animal.py
class Animal:
    
    def __init__(self, especie, nombreMascota, edad):
        self.especie = especie
        self.nombreMascota = nombreMascota
        self.edad = edad
        
    def getNombreMascota(self):
        return self.nombreMascota
    
    def setNombreMascota(self, nombreMascota):
        self.nombreMascota=nombreMascota
        
    def getEspecie(self):
        return self.nombreMascota
    
    def setEspecie(self, especie):
        self.especie=especie
        
    def getEdad(self):
        return self.edad
    
    def setEdad(self, edad):
        self.edad=edad